services/tenant-ui/frontend/src/components/contacts/Contacts.vue

```diff
@@ -105,6 +126,11 @@ const doDelete = (schema: any) => {
       toast.error(`Failure: ${err}`);
     });
 };
+
+const expandedRows = ref([]);
+const onRowExpand = async (event: any) => {
+  await getFullContact(event.data.contact_id);
+};
```

Comment:
Can we pass the composable down into the expand component? So some of the generic work is being done there instead of in each table component.
Like the component knows it's opened (onMount maybe...) and then it invokes the supplied method to get data?

Reply:
Then you don't have to pass in loading and data, the sub component would have that stuff encapsulated in it's own scope? I haven't thought it through entirely so could be some stopper for that but a thought.

Reply:
was working on that. it needs to be in a sub-component anyway, or else every row is binding to the same item (one referenced in Contacts.vue.  So there will be a wrapper component that will fetch it's own data then make use of expand row...

Reply:
now have examples for contact, schema and verifier presentation. those are self contained (ie. the table component doesn't do anything around fetching data...)